plugins/woocommerce/includes/class-wc-form-handler.php

```diff
@@ -206,6 +206,14 @@ public static function save_address() {
 
 		wc_add_notice( __( 'Address changed successfully.', 'woocommerce' ) );
 
+		/**
+		 * Hook: woocommerce_customer_save_address.
+		 *
+		 * Fires when an address is edited via a customer's "My account" screen.
+		 *
+		 * @param int    $user_id User ID being saved.
+		 * @param string $load_address Type of address e.g. billing or shipping.
+		 */
```

Comment:
Thanks for adding this, however a docblock has already been added to this location.